### DISCUSSION TITLE
Ghcr.io images are not accessible via UI when uploaded via Actions

### DISCUSSION THREAD

**Discussion by plutzilla** [community/community#26020] [Actions]
<p>Hi,</p>
<p>I use the following action to upload the Docker image to the <a href="http://ghcr.io" rel="nofollow noopener">ghcr.io</a> registry:</p>
<pre><code class="lang-auto">      - name: Push to GitHub Container Registry
        uses: docker/build-push-action@v1
        with:
          dockerfile: nginx.Dockerfile
          username: ${{ github.actor }}
          password: ${{ secrets.GHCR_PAT }}
          registry: ghcr.io
          repository: plutzilla/lescinskas.lt-website
          tag_with_ref: true
</code></pre>
<p>The image is pushed (and can be pulled using the same PAT), but cannot be accessed via UI:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/5a8b075da61ef03be62092a3e903381f3939d26e.png" data-download-href="/uploads/short-url/cUYQ5oglxiKo31ftGYpJV0kgE1g.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1925785/181094786-566c2fcb-0273-4f2d-ad3a-20f597e55c74.png" alt="image" data-base62-sha1="cUYQ5oglxiKo31ftGYpJV0kgE1g" width="690" height="75" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/5a8b075da61ef03be62092a3e903381f3939d26e_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">747×82 6.73 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Hence, it is not possible to manage the package (set permissions, delete etc.).</p>
<p>FYI, authenticated GraphQL query does not return GHCR packages at all:</p>
<pre><code class="lang-auto">query {
  user(login: "plutzilla") {
    packages(first: 10) {
      nodes {
        name
        latestVersion { version }
      }
    }
  }
}
</code></pre>
<p>returns only repository-based packages.</p>

**Comment by NicoNBW** [comment]
<p>Thanks for your feedback.<br>
Could you please check the workflow log to confirm again whether you pushed this image successfully?<br>
If you have successfully pushed, please click your profile photo in the top right corner of GitHub, then click  <strong>Your profile</strong> &gt;&gt; <strong>Packages</strong> to check if your image exists in the packages list.<br>
More about Managing container images with GitHub Container Registry please refer <a href="https://docs.github.com/en/packages/managing-container-images-with-github-container-registry" rel="nofollow noopener">here</a>.</p>

**Comment by plutzilla** [comment]
<p>Hi <a class="mention" href="/niconbw">@niconbw</a>,<br>
thanks for your response.</p>
<p>I can see the GHCR packages in the profile package list, however the package count is wrong in the top navigation bar and in the package list:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/432091188762a05d147733c8d4787e6d9cf364f5.png" data-download-href="/uploads/short-url/9zPLoPQPLE0FqVmHoQYZLPy8pjT.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1925785/181094792-99b1487c-e82d-47a5-82e0-1b2078c93e6f.png" alt="image" data-base62-sha1="9zPLoPQPLE0FqVmHoQYZLPy8pjT" width="690" height="348" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/432091188762a05d147733c8d4787e6d9cf364f5_2_690x348.png, https://aws1.discourse-cdn.com/github/original/2X/4/432091188762a05d147733c8d4787e6d9cf364f5.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/4/432091188762a05d147733c8d4787e6d9cf364f5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/432091188762a05d147733c8d4787e6d9cf364f5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">921×465 40 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The images are built and pushed successfully via Actions. See the log excerpt:</p>
<pre><code class="lang-auto">...
Successfully built 29035bbfa85e
Successfully tagged ghcr.io/plutzilla/lescinskas.lt-website:cicd-with-actions
Pushing image [ghcr.io/plutzilla/lescinskas.lt-website:cicd-with-actions]
The push refers to repository [ghcr.io/plutzilla/lescinskas.lt-website]
c83e79eb5d7b: Preparing
550333325e31: Preparing
22ea89b1a816: Preparing
a4d893caa5c9: Preparing
0338db614b95: Preparing
d0f104dc0a1f: Preparing
d0f104dc0a1f: Waiting
550333325e31: Layer already exists
22ea89b1a816: Layer already exists
0338db614b95: Layer already exists
a4d893caa5c9: Layer already exists
d0f104dc0a1f: Layer already exists
c83e79eb5d7b: Pushed
cicd-with-actions: digest: sha256:ccff69dca2784486e3ae4d2a61556f00056a5793d306f80438dd972c4e03a107 size: 1573
</code></pre>
<p>Clicking on the package in the package list returns 404 page.</p>
<p>The package <code>ghcr.io/plutzilla/test</code> which pas pushed from desktop, is accessible via UI.</p>

**Comment by NicoNBW** [comment, marked as answer]
<p>Hi <a class="mention" href="/plutzilla">@plutzilla</a><br>
About package count issue:<br>
I found <a href="https://github.community/t/github-container-registry-ghcr-io-packages-not-appearing-in-webinterface/130077/4">a similar post</a> mentioning this issue, and the GitHub engineering team is already aware of this, you can keep track of this issue via the link above.</p>
<p>About 404 error:<br>
Do you also get a 404 error when you click on your first image(lescinskas.It) in your screenshot?<br>
Tested in my side, I found that if the package name contains dot(.) character, it will cause this problem. You could try other names that do not contain a dot.<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/bf1a71a400ccd7f7b2ab56a6cfc3216af8bbb50b.png" data-download-href="/uploads/short-url/rgzQRESSPGwNZNRQKfQaIFyUfzl.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46212442/181094793-b115e085-2592-4a8a-8392-99b744bb081f.png" alt="image" data-base62-sha1="rgzQRESSPGwNZNRQKfQaIFyUfzl" width="690" height="189" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/bf1a71a400ccd7f7b2ab56a6cfc3216af8bbb50b_2_690x189.png, https://aws1.discourse-cdn.com/github/original/2X/b/bf1a71a400ccd7f7b2ab56a6cfc3216af8bbb50b.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/bf1a71a400ccd7f7b2ab56a6cfc3216af8bbb50b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/bf1a71a400ccd7f7b2ab56a6cfc3216af8bbb50b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1019×280 15.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by plutzilla** [comment]
<p>Thank you. The issue is indeed in the dot character in the image name - not related to the Github Actions.</p>

**Comment by plutzilla** [comment]
<p>FYI created the topic in the Github Packages category: <a href="https://github.community/t/ghcr-images-having-dot-in-the-name-are-not-accessible-in-ui/130390" class="inline-onebox">GHCR images having dot in the name are not accessible in UI</a></p>

**Comment by NicoNBW** [comment]
<p>Hi <a class="mention" href="/plutzilla">@plutzilla</a><br>
Thanks for your update. I will follow this issue with you through the new topic in the Github Packages category. Hope it will be resolved soon.</p>
